clients/hadoopfs/src/test/java/io/lakefs/LakeFSFileSystemTest.java

```diff
@@ -198,6 +196,73 @@ public void testExists_NotExists() throws ApiException, IOException {
 
         Assert.assertFalse(fs.exists(p));
     }
+    
+    @Test
+    public void testDelete_FileExists() throws ApiException, IOException {
+        when(objectsApi.statObject(eq("repo"), eq("main"), eq("delete/sample/file.txt")))
+                .thenReturn(new ObjectStats().
+                        path("lakefs://repo/main/delete/sample/file.txt").
+                        pathType(PathTypeEnum.OBJECT).
+                        physicalAddress(s3Url("delete/sample/file.txt")).
+                        checksum("unused").
+                        mtime(0L).
+                        sizeBytes(42L));
```

Comment:
@talSofer names these constants in #2014.  Can we use those here, too?  (Alternatively, I don't mind ditching the constants in favour of these values, as long as we do _that_ consistently.)

Reply:
reused the constants